suites/apis/requestorNewTest.js

```diff
@@ -11,15 +11,33 @@ Feature('RequestorNewAPI @requires-requestorNew');
 const { expect } = require('chai');
 const requestorTasks = require('../../services/apis/requestor/requestorTasks.js');
 
+BeforeSuite(async ({
+  I,
+}) => {
+  I.cache = {};
+  I.cache.requestDidList = [];
+});
+
 Before(async ({
   home, users, login,
 }) => {
   home.do.goToHomepage();
   login.complete.login(users.mainAcct);
 });
 
+AfterSuite(async ({ I }) => {
+  console.log('Deleting all the requests created by this test with user cdis.autotest@gmail.com...');
+  console.log(I.cache.requestDidList);
```

Comment:
can we add a check to print these values only in debug mode

Reply:
oh yes, I can do that